components/raftstore/src/coprocessor/split_observer.rs

```diff
@@ -23,24 +22,12 @@ impl SplitObserver {
             return Err("key is empty".to_owned());
         }
 
-        let mut key = match bytes::decode_bytes(&mut key.as_slice(), false) {
+        let key = match bytes::decode_bytes(&mut key.as_slice(), false) {
```

Comment:
Why this is changed?

Reply:
It's an obsolete code, back then we store each column on its own key, column id is appended at the end of the key, we need to make sure a row stores in a single region.